.github/no-response.yml

```diff
@@ -0,0 +1,12 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an issue is closed for lack of response
+daysUntilClose: 7
+
+# Label requiring a response
+responseRequiredLabel: Pending Feedback
```

Comment:
Can we make it _skip_ issues that have that label instead of requiring that label to be activated?

Reply:
@yousefhamza unfortunately we can't do that. [Refer](https://github.com/lee-dohm/no-response)

Reply:
Can you screenshot/link which part of the document mentioning this? I'm assuming you can just remove this line `responseRequiredLabel: Pending Feedback`